examples/svelte/simple/src/App.svelte

```diff
@@ -0,0 +1,150 @@
+        <input
+          id={field.name}
+          type="text"
+          placeholder="First Name"
+          value={field.state.value}
+          onblur={() => field.handleBlur()}
+          oninput={(e: Event) => {
+            const target = e.target as HTMLInputElement
+            field.handleChange(target.value)
+          }}
+        />
+        <FieldInfo {field} />
+      </div>
+    {/snippet}
+  </form.Field>
+  <form.Field
+    name="lastName"
+    validators={{
+      onChange: ({ value }) =>
+        value.length < 3 ? 'Not long enough' : undefined,
+    }}
+  >
+    {#snippet children(field)}
+      <div>
+        <label for={field.name}>Last Name</label>
+        <input
+          id={field.name}
+          type="text"
+          placeholder="Last Name"
+          value={field.state.value}
+          onblur={() => field.handleBlur()}
+          oninput={(e: Event) => {
+            const target = e.target as HTMLInputElement
+            field.handleChange(target.value)
+          }}
+        />
+        <FieldInfo {field} />
+      </div>
+    {/snippet}
+  </form.Field>
+  <form.Field name="employed">
+    {#snippet children(field)}
+      <div>
+        <label for={field.name}>Employed?</label>
+        <input
+          oninput={() => field.handleChange(!field.state.value)}
+          checked={field.state.value}
+          onblur={() => field.handleBlur()}
+          id={field.name}
+          type="checkbox"
+        />
+      </div>
+      {#if field.state.value}
+        <form.Field
+          name="jobTitle"
+          validators={{
+            onChange: ({ value }) =>
+              value.length === 0 ? 'If you have a job, you need a title' : null,
+          }}
+        >
+          {#snippet children(field)}
+            <div>
+              <label for={field.name}>Job Title</label>
+              <input
+                type="text"
+                id={field.name}
+                placeholder="Job Title"
+                value={field.state.value}
+                onblur={field.handleBlur}
+                oninput={(e: Event) => {
+                  const target = e.target as HTMLInputElement
+                  field.handleChange(target.value)
+                }}
+              />
+              <FieldInfo {field} />
+            </div>
+          {/snippet}
+        </form.Field>
+      {/if}
+    {/snippet}
+  </form.Field>
+  <div>
+    <form.Subscribe
+      selector={(state) => ({
+        canSubmit: state.canSubmit,
+        isSubmitting: state.isSubmitting,
+      })}
+    >
+      {#snippet children({ canSubmit, isSubmitting })}
+        <button type="submit" disabled={!canSubmit}>
+          {isSubmitting ? 'Submitting' : 'Submit'}
+        </button>
+      {/snippet}
+    </form.Subscribe>
+    <button
+      type="button"
+      id="reset"
+      onclick={() => {
+        form.reset()
+      }}
+    >
+      Reset
+    </button>
+  </div>
+</form>
+
+<pre>{JSON.stringify(form.state, null, 2)}</pre>
```

Comment:
This doesn't seem to be reactive

Reply:
This is a leftover from some early explorations - I just removed it. FWIW for Solid and Vue this is also unreactive, from what I understand you gotta use `form.useStore(...)` to get a reactive version of (a part of) the state.